webview-resources/document.css

```diff
@@ -0,0 +1,19 @@
+.btn {
+	border: 0;
+	color: var(--vscode-button-foreground);
+	background-color: var(--vscode-button-background);
+}
+
+.btn svg {
+	fill: var(--vscode-button-foreground);
+}
+
+.btn:hover {
+	background-color: var(--vscode-button-hoverBackground);
+}
+
```

Comment:
Style `.btn:focus` to remove the borders from the "back to top" button after its clicked:

![ButtonHighlightingMarkdownDocumentation](https://user-images.githubusercontent.com/22376627/93351446-c3b37900-f807-11ea-8d89-39695e9f74a0.png)

(to be consistent with other buttons like this in the UI such as the git initialize a repository button).

Reply:
What dif you suggested  for  `.btn:focus`  style?

Reply:
Let me try some things out quickly to see. I think something along the lines of this should work:
```css
.btn:focus {
  border: 0px;
}
```

Reply:
I have copied/pasted this CSS from vscode-java. I suggest that you create an issue about that and when we will find a clean solution we could fix it and vscode-java too.

Reply:
Opened #330

Reply:
Thanks @datho7561